content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
++++
+title = "Bevy 0.6"
+date = 2022-01-06
+[extra]
+author = "Carter Anderson"
+twitter = "cart_cart"
+github = "cart"
+youtube = "cartdev"
+image = "bistro_night.png"
+show_image = true
+image_subtitle = "The Lumberyard Bistro scene rendered in the New Bevy Renderer by @mockersf"
+image_subtitle_link = ""
++++
+
+Thanks to **X** contributors, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.6** release on [crates.io](https://crates.io/crates/bevy)!
+
+For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out [Quick Start Guide](/learn/book/getting-started/) to get started. Bevy is also free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
+
+To update an existing Bevy App or Plugin to **Bevy 0.6**, check out our [0.5 to 0.6 Migration Guide](/learn/book/migration-guides/0.5-0.6/).
+
+There are a _ton_ of improvements in this release, but here are some of the big ones: 
+
+* A brand new modern renderer that is prettier, faster, and easier to extend
+* Directional and point light shadows
+* Clustered forward rendering
+* Frustum culling
+* Significantly faster sprite rendering with less boilerplate
+* Native WebGL2 support
+* High level custom Materials
+* More powerful shaders: preprocessors, imports, WGSL support
+* Bevy ECS ergonomics and performance improvements. No more `.system()`!
+
+Read on for details!
+
+<!-- more -->
+
+## The New Bevy Renderer
+
+**Bevy 0.6** introduces a brand new modern renderer that is:
+* **Faster**: More parallel, less computation per-entity, more efficient CPU->GPU dataflow, and (soon-to-be-enabled) pipelined rendering
+* **Simpler**: Fewer layers of abstraction, simpler data flow, improved low-level, mid-level, and high-level interfaces, direct wgpu access 
+* **Modular to its core**: Standardized 2d and 3d core pipelines, extensible Render Phases and Views, composable entity/component-driven draw functions, shader imports, extensible and repeatable render pipelines via "sub graphs"
+* **Prettier**: We're releasing the new renderer alongside a number of graphical improvements, such as directional and point light shadows, clustered forward rendering (so you can draw more lights in a scene), and spherical area lights. We also have a ton of new features in development (cascaded shadow maps, bloom, particles, shadow filters, and more!)
```

Comment:
I dig it! I chose not to add credits to this section because I think it distracts from the "snappyness" of the messaging (note that I didn't credit myself here either ... I just used "we"). Rob gets ample credits in the rest of the blog post. I don't think that message will be lost.